Makefile

```diff
@@ -76,9 +76,10 @@ down: down-go-quorum down-besu
 
 down-dev:
 	@docker-compose -f ./docker-compose.yml down --volumes --timeout 0
+	@make down-deps
```

Comment:
lol you keep adding this and I keep removing it. I don't want my deps to go down when I take down my qkm. Maybe we can add a new command like `up-qkm` ?

Reply:
Actually I was wondering why this was missing I was almost 100% sure I added xD. 

Since we require `deps` to be up for running `make dev` I think we should exact the oppositive when you do `make down`